community-generators/prisma-json-schema-generator/.gitignore

```diff
@@ -0,0 +1 @@
+json-schema/json-schema.json
```

Comment:
Any negative side effects of removing these files from git and gitignoring them? We compare via snapshots...

Reply:
I think that's fine to remove